.env.ci

```diff
@@ -41,7 +41,7 @@ GEETEST_ID=${GEETEST_ID}
 GEETEST_KEY=${GEETEST_KEY}
 TWILIO_ACCOUNT_SID=${TWILIO_ACCOUNT_SID}
 TWILIO_AUTH_TOKEN=${TWILIO_AUTH_TOKEN}
-TWILIO_PHONE_NUMBER=${TWILIO_PHONE_NUMBER}
+TWILIO_VERIFY_SERVICE=${TWILIO_VERIFY_SERVICE}
```

Comment:
Have the same comment here as on the [org-infra PR](https://github.com/GaloyMoney/galoy-org-infra/pull/107#pullrequestreview-1221966278) regarding the name of the variable